suite-native/device-manager/src/components/DeviceList.tsx

```diff
@@ -140,42 +137,42 @@ export const DeviceList = ({ isVisible, onSelectDevice }: DeviceListProps) => {
         return null;
     }
 
+    const hasNotSelectedDevicesToShow = notSelectedInstancelessDevices.length > 0;
+
     return (
         <Animated.View style={[listAnimatedStyle, applyStyle(listStaticStyle)]}>
-            <VStack>
-                <Box>
-                    {notSelectedInstancelessDevices.map(d => {
-                        if (d.state === undefined) {
-                            return null;
-                        }
-
-                        return (
-                            <DeviceItem
-                                key={d.state}
-                                deviceState={d.state}
-                                onPress={() => onSelectDevice(d)}
-                            />
-                        );
-                    })}
-                </Box>
-
-                {A.isEmpty(notSelectedInstancelessDevices) ? (
+            <Box>
+                {notSelectedInstancelessDevices.map(d => {
+                    if (d.state === undefined) {
+                        return null;
+                    }
+
+                    return (
+                        <DeviceItem
```

Comment:
nit: Could be simplified to omit the nullish check at the beginning:
```suggestion
                        {d.state & <DeviceItem
```

Reply:
f312d14b01